### ISSUE TITLE
Update 'notify_team_new_comment' GitHub action to ignore comments by the Sentry bot

### ISSUE THREAD

**Issue by MisRob** [learningequality/kolibri#12568]
## Summary

Update 'notify_team_new_comment' GitHub action to ignore comments by the Sentry bot

## Observed behavior

The GitHub action that sends a Slack notification when a contributor comments on issue, [.github/workflows/notify_team_new_comment.yml](https://github.com/learningequality/kolibri/blob/develop/.github/workflows/notify_team_new_comment.yml), sends us notifications for comments by the Sentry bot `sentry-io[bot]`:

![Screenshot from 2024-08-16 19-54-33](https://github.com/user-attachments/assets/3097add4-696b-4404-858e-313ee5bdfa76)

Related comment: https://github.com/learningequality/studio/issues/4643#issuecomment-2291352676

## Expected behavior

`notify_team_new_comment` is updated so that we don't receive notifications from `sentry-io[bot]`.

## Comments

Unless https://github.com/learningequality/kolibri/issues/12567 is resolved at first, note that copies of this action need to be updated in the same way in the following repositories:
- Studio
- Kolibri Design System
  - Even though strictly speaking we're not receiving comments from Sentry there, it'll be good to keep them consistent because of the  plan to make it into a re-usable action

**Comment by josephinoo** [comment]
Hi @MisRob MisRob! 👋 Is this issue open for contributions?

**Comment by MisRob** [comment]
Hi @josephinoo, I'm happy to assign this to you. I'm not exactly sure how to identify that the author is Sentry bot. Would you be able to research it and test it?

**Comment by josephinoo** [comment]
Thanks, @MisRob , I'll look into it and if I find anything I'll let you know.

**Comment by MisRob** [comment]
Hi @josephinoo, I've just realized we will also need to ignore comments by Learning Equality bot such as this one

![Image](https://github.com/user-attachments/assets/7a7525c1-0a5c-4cde-8af1-90f5be01c57a)

would you like to include it in this issue, or should I open a new one?

Meanwhile I realized there should be a simple way to figure out what's the bot's name, since we actually show it exactly as it is in the Slack message. Code for it is `github.event.comment.user.login` ([reference](https://github.com/learningequality/.github/blob/main/.github/workflows/notify_team_new_comment.yml#L34)) so I think just comparing that to "sentry-io[bot]" and "learning-equality-bot[bot]" in a condition should do.

**Comment by Sanchit-100** [comment]
Hi @MisRob .
Hi @josephinoo . I would like to know if you are working on this issue.
If you are busy otherwise, I would like to get this assigned.
Thanks

**Comment by MisRob** [comment]
Hi @Sanchit-100, since we didn't hear back from @josephinoo here and on other issues for several weeks, I will re-assign. Please see the comments above with the additional guidance.

**Comment by MisRob** [comment]
Thank you

**Comment by iamshobhraj** [comment]
Hi @Sanchit-100 , are you working on this issue?

**Comment by Sanchit-100** [comment]
Hey @iamshobhraj.
Yes, I am currently working on this issue.
Will soon post the updates.
Thanks

**Comment by MisRob** [comment]
Hi @Sanchit-100, due to some new actions that have dependency on having this issue merged, I will need to increase priority here and take care of it rather sooner. I wanted to check in with you on how things look like, would you please let me know?

**Comment by yashhash2** [comment]
Hey @Sanchit-100 are you working on this issue if not then @MisRob can assign this to me. 
If you are working all the best :)

**Comment by Sanchit-100** [comment]
Hi @MisRob 
I have created a PR to resolve the issue.
PR link: https://github.com/learningequality/.github/pull/19
Kindly review and let me know in case any change is required.

**Comment by Sanchit-100** [comment]
The [notify_team_new_comment.yml](https://github.com/learningequality/kolibri/blob/develop/.github/workflows/notify_team_new_comment.yml) file in kolibri repo calls the workflow from [.github repo](https://github.com/learningequality/.github/blob/main/.github/workflows/notify_team_new_comment.yml).
Thus, to resolve the issue centrally across all platforms, PR was raised in the .github repo's workflow.

**Comment by MisRob** [comment]
Yes, that makes sense @Sanchit-100. Thank you for following-up! I will review.

**Comment by MisRob** [comment]
Closed by https://github.com/learningequality/.github/pull/19
